code/workspaces/web-app/src/auth/authConfig.js

```diff
@@ -9,10 +9,6 @@ const getAuthConfig = () => axios
         ...data,
         silent_redirect_uri: `${window.location.origin}/silent_callback`,
         redirect_uri: `${window.location.origin}/callback`,
-        metadata: {
-          ...data.metadata,
-          end_session_endpoint: `${data.authority}/v2/logout?returnTo=${encodeURIComponent(window.location.origin)}&client_id=${data.client_id}`,
-        },
```

Comment:
This causes my logout on my local environment to go to https://testlab.datalabs.localhost/ instead of http://localhost:3000/.

Reply:
Ah I see don't you use testlab.datalabs.localhost anyway? You can still make this work then but would need to manually edit the `end_session_endpoint` in the development version of `web_auth_config`. The issue is with keeping it as it is is that it will only ever work locally for auth0.